prime-router/src/main/kotlin/transport/EmailTransport.kt

```diff
@@ -0,0 +1,79 @@
+package gov.cdc.prime.router.transport
```

Comment:
Hmm.  Technically, this isn't really a 'Transport' since nothing's getting transported.    The transport mechanism is like "make the file available for download" or something like that? 

And sending an email is a generic activity - can we genericize this code, make it static, move out the 'from' and 'subject' (which I'm guessing you were intending to do anyways) and just have a general emailing facility that can contact anybody about anything?